src/main/java/org/openstreetmap/atlas/tags/FordTag.java

```diff
@@ -0,0 +1,30 @@
+package org.openstreetmap.atlas.tags;
+
+import org.openstreetmap.atlas.tags.annotations.Tag;
+import org.openstreetmap.atlas.tags.annotations.TagKey;
+import org.openstreetmap.atlas.tags.annotations.validation.Validators;
+
+/**
+ * OSM Ford Tag.
+ *
+ * @author sayas01
+ */
+@Tag(taginfo = "https://taginfo.openstreetmap.org/keys/ford#values", osm = "https://wiki.openstreetmap.org/wiki/Key:ford")
+public enum FordTag
+{
+    YES,
+    STEPPING_STONES,
+    SEASONAL,
+    NO,
+    STREAM,
+    LEVEL_CROSSING,
+    BOAT;
+
+    @TagKey
+    public static final String KEY = "ford";
+
+    public static boolean isYes(final Taggable taggable)
```

Comment:
Is this function necessary?

Reply:
I added this function because tag value "yes" constitute about 98.45% of all objects with ford key and it is this condition that we are incorporating in the check's enhancement. I referred other tag classes and added this. What do you think?

Reply:
Yeah, great reason!